src/ConstructiveSolidGeometry/Transformations.jl

```diff
@@ -1,20 +1,20 @@
-struct StretchedGeometry{T,P<:AbstractGeometry{T}} <: AbstractGeometry{T}
+struct ScaledGeometry{T,P<:AbstractGeometry{T}} <: AbstractGeometry{T}
     p::P
     inv_s::SVector{3,T}
-    StretchedGeometry(p::P, s::SVector{3,T}) where {T,P} = new{T,P}(p, inv.(s))
+    ScaledGeometry(p::P, s::SVector{3,T}) where {T,P} = new{T,P}(p, inv.(s))
 end
-in(p::CartesianPoint, g::StretchedGeometry) = in(CartesianPoint(g.inv_s .* p), g.p)
-stretch(g::AbstractGeometry, s::SVector{3,T}) where {T} = StretchedGeometry(g, s)
-stretch(g::StretchedGeometry, s::SVector{3,T}) where {T} = StretchedGeometry(g.p, g.inv_s .* inv.(s))
+in(p::CartesianPoint, g::ScaledGeometry) = in(CartesianPoint(g.inv_s .* p), g.p)
+scale(g::AbstractGeometry, s::SVector{3,T}) where {T} = ScaledGeometry(g, s)
+scale(g::ScaledGeometry, s::SVector{3,T}) where {T} = ScaledGeometry(g.p, g.inv_s .* inv.(s))
```

Comment:
Shouldn't this be `ScaledGeometry(g.p, inv.(g.inv_s) .* s)`? :thinking: 
or better: `ScaledGeometry{T}(g.p, g.inv_s) .* inv.(s))`

Reply:
Good catch! Calling `scale` on a `ScaledGeometry` returns a wrong `inv_s`!